### PR TITLE
Fix live tests failures

### PR DESCRIPTION
**Pull request by kryalama**
Failures:
Live tests were failing due to auto redirects from ingestion endpoint. Also there was not 307 redirect support.
Due to usage of HttpPolicyProviders GlobalOpentelemetry object was being initialized twice
Live tests based on AAD authentication were failing. Will address this issue in a separate PR

### CODE REVIEW

**Comment by trask** [comment]
@kryalama can you summarize what the issue was? thx!

**Comment by kryalama** [comment]
> @kryalama can you summarize what the issue was? thx!

Updated the description, thanks

File: sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/MonitorExporterClientTestBase.java [hunk 1]
```diff
@@ -39,40 +38,17 @@ AzureMonitorExporterBuilder getClientBuilder() {
             httpClient = interceptorManager.getPlaybackClient();
         }
 
+        List<HttpPipelinePolicy> policies = new ArrayList<>();
+        policies.add(new AzureMonitorRedirectPolicy());
+        policies.add(interceptorManager.getRecordPolicy());
         HttpPipeline httpPipeline = new HttpPipelineBuilder()
             .httpClient(httpClient)
-            .policies(interceptorManager.getRecordPolicy()).build();
+            .policies(policies.toArray(new HttpPipelinePolicy[0]))
```
Comment (trask) [inline sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/MonitorExporterClientTestBase.java:46]: instead of creating policies list above
```suggestion
            .policies(new AzureMonitorRedirectPolicy(), interceptorManager.getRecordPolicy());
```

**Review by trask — APPROVED** [review]
